services/slack/intermediate.go

```diff
@@ -117,6 +118,11 @@ func (t *Transformer) TransformUsers(users []SlackUser) {
 
 	resultUsers := map[string]*IntermediateUser{}
 	for _, user := range users {
+		var deleteAt int64 = 0
+		if user.Deleted {
+			deleteAt = model.GetMillis()
```

Comment:
If a proper timestamp is needed, maybe we can use the `updated` field in the user model (if that's provided in the export). Just a comment, IMO is fine as it is.

Reply:
Since there's no way of knowing specifically when the user was deactivated on Slack's side, I'm thinking we should treat the "sync time" (in this case at the time of transforming the import, as we are in this PR) as the time we have "detected they are deactivated". 0/5 on this though